content/release.rst

```diff
@@ -197,3 +210,78 @@ Sample release notes
     * Fix an internal server error when FTP client requests
       an unknown command. [#160][ftp][ftps]
 
+
+Version Management
+==================
+
+Chevah release versions are based on MAJOR.MINOR.PATCH
+`Semantic Versioning specification <http://semver.org/>`_.
+
+A MAJOR version is released to introduce new features,
+remove functionalities which have become obsolete, or add features not
+compatible with previous versions.
+
+MINOR versions are released based on a rolling update development model at
+intervals varying between 30 to 60 days.
+The goal is to have functionalities and defect fixes available to customers as
+soon as possible.
+Each release has a certain overhead, and the overhead should be minimize by
+automating the release process.
+In an ideal works a release should be done by preparing a release branch
+and then by issuing a single command the documentation, download and news pages
+are updated, while users are automatically notified about the new releases.
+
+PATCH versions are released as soon as a defect is fixed,
+usually one week after it has been initially discovered and reported.
+Security issues have top priority and a fix is released as soon as possible.
+
+
+Compatibility Policy
+====================
+
+Any release inside a MAJOR version release series should be backward and
+forward compatible with any other release.
+
+That is, users should be able to upgrade or downgrade to any minor release
+without having to change any external system interaction, API interaction or
+configuration option.
+
+Some MINOR version might introduce various functionalities which are not
+available in previous versions. Downgrading to a previous MINOR version will
+not make the newest functionalities available but configuration options or
```

Comment:
Comma required before _but_ as it joins two stand-alone sentences. https://trac.chevah.com:10443/wiki/Development/EnglishForRomanians#comma-before-conjunctions